python_cmake_module/package.xml

```diff
@@ -10,7 +10,7 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <buildtool_export_depend>python3-dev</buildtool_export_depend>
+  <buildtool_export_depend>python3</buildtool_export_depend>
```

Comment:
I don't know why this package would need `python3-dev` but the package `rosidl_generator_py` in this repo is building a Python extension and therefore needs to have that line: `<buildtool_export_depend>python3</buildtool_export_depend>`